src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -183,10 +196,14 @@ private boolean lineMatching(int currentLine) {
     private boolean isMatchingByContextStrategy(TreeWalkerAuditEvent event) {
         boolean result = false;
         if (containsShortName(supportContextStrategyChecks, event)
-                || containsShortName(checkNamesForContextStrategyByTokenOrParentSet, event)) {
+                || containsShortName(checkNamesForContextStrategyByTokenOrAncestorSet, event)) {
             final DetailAST eventAst;
-            if (containsShortName(checkNamesForContextStrategyByTokenOrParentSet, event)) {
-                eventAst = getEventAst(event).getParent();
+            if (containsShortName(checkNamesForContextStrategyByTokenOrAncestorSet, event)) {
+                DetailAST tempAst = getEventAst(event).getParent();
```

Comment:
you don't need new variable, just use `eventAst`

Reply:
done